experimental/traffic-portal/src/app/core/certs/cert-detail/cert-detail.component.html

```diff
@@ -0,0 +1,68 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div>
+	<div class="mat-elevation-z2">
+		<h2>Issuer</h2>
+		<tp-cert-author [author]="issuer"></tp-cert-author>
+	</div>
+	<div class="mat-elevation-z2">
+		<h2>Subject</h2>
+		<tp-cert-author [author]="subject"></tp-cert-author>
+	</div>
+	<div class="mat-elevation-z2">
+		<h2>Validity</h2>
+		<div class="content">
+			<div class="double">
+				<mat-form-field>
+					<mat-label>Not Valid Before</mat-label>
+					<input matInput type="date" [formControl]="validBeforeFormControl" name="validBefore" readonly />
```

Comment:
just checking: is this always representative of a day, and never include a time?